.github/workflows/maven.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
           git config --global user.email "github-actions@github.com"
           git config --global user.name "GitHub Actions"
           mvn release:prepare -B -DreleaseVersion=${{ inputs.release-version }} -DdevelopmentVersion=${{ inputs.next-version }} -DpushChanges=false -DremoteTagging=false
-          mvn release:perform -Darguments="-Dmaven.deploy.skip=true" -DlocalCheckout=true
+          mvn release:perform -DlocalCheckout=true
```

Comment:
ich würde das auch nicht mehr so machen sondern gleich nach oben verschieben. in den release:prepare schritt.

Reply:
bitte dann auch gleich in unseren Templates ändern. weil sonst muss ich jetzt alles doppelt reviewn

Reply:
@hupling Since this PR is already merged, please provide a new Issue or PR with your suggested changes. :)